### ISSUE TITLE
2024.09.13 개인미팅 피드백 내용

### ISSUE THREAD

**Issue by hsh200315** [hsh200315/ELL_Intern#1]
### 코드 리뷰

1. 들여쓰기 tab이 아닌 공백(space) 4칸 사용. 들여쓰기(탭/space) 일치 필요
2. nn.Module 공부 이후, class 내 함수 선언이 아닌 BasicBlock(64, 64)와 같이 선언하여 layer 생성할 수 있도록 수정
3. ResNet에서, 원래 block안에서 addition -> ReLU까지 진행. 이러하게 수정
4. ResNet에서 MaxPool2d 하지말고 32*32 input 이후 kernel:3, padding:1, stride:1 conv연산을 통해 진행
5. ResNet 현재 layer 수마다 구현되어 있는데 통합 진행
6. Tensorboard에 epoch마다 Accuracy 추가

**Comment by hsh200315** [comment]
1. 완료
2. 완료
3. 완료
4. 완료
5. 완료
